config/php-cs-fixer.php

```diff
@@ -40,7 +40,6 @@
             'yoda_style' => ['equal' => false, 'identical' => false, 'less_and_greater' => false],
 
             // function notation
-            'native_function_invocation' => ['include' => ['@all']],
```

Comment:
I assume that PHP-CS-Fixer is not smart enough to see where functions are being `use`d.

Reply:
Correct.